src/ApiCompat/Microsoft.DotNet.ApiCompat.Task/MSBuildCompatibilityLogger.cs

```diff
@@ -38,9 +40,13 @@ private bool LogSuppressableMessage(MessageLevel messageLevel, Suppression suppr
             if (_suppressionEngine.IsErrorSuppressed(suppression))
                 return false;
 
+            _suppressionWasLogged = true;
+
             _log.Log(new Message(messageLevel, string.Format(format, args), code));
 
             return true;
         }
+
+        public bool SuppressionWasLogged => _suppressionWasLogged;
```

Comment:
```suggestion
        public bool SuppressionWasLogged { get; private set; }
```

Reply:
You don't need to save a state for the msbuild logger as it already has a property on the `_log` field, called `HasLoggedAnyErrors` (or similar).